documentation/examples/dynamic-sync-table/README.md

```diff
@@ -1,11 +1,3 @@
-Most sync tables have schemas that can be statically defined. For example, if you're writing a sync of a user's Google Calendar events, the structure of an Event from the Google Calendar API is well-known and you can write a schema for what your table should contain.
+Dynamic sync tables allow you to bring data from an external data source into your doc, even when the structure of that data is not known in advance. Instead of including a static schema in your sync table definition, you include a `getSchema` method that returns a schema based on the dataset the user selects. It can use the fetcher to make authenticated HTTP requests to an API so that you can determine the shape of the data.
```

Comment:
This still uses "method", was that intentional?

Reply:
No, overlooked. Fixed.